### PR TITLE
Add nl-parser

### PR DESCRIPTION
**Pull request by ashwathishiva**

### CODE REVIEW

File: manifests/base/resources/nl-parser/transformers/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  - ../../../transformers
+transformers:
+- ../../../../../.operator/transformers
+- ../../../../../configuration/transformers
+#Disable all transformers except these
+patches:
+- target:
+    labelSelector: >-
+      key notin ( 
+        dpsUri
```
Comment (ffoysal) [inline manifests/base/resources/nl-parser/transformers/kustomization.yaml:16]: `ingressClass` transformer needed

File: manifests/base/resources/nl-parser/patches/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  template:
+    metadata:
+      labels:
+        nl-parser-nats-client: null
+    spec:
+      containers:
+      - name: main
+        imagePullPolicy: IfNotPresent
+        env:
+        - name: DPS_URI
```
Comment (ffoysal) [inline manifests/base/resources/nl-parser/patches/deployment.yaml:21]: need to delete the old value first ?
```
- $patch: delete
   name: DPS_URI
```

File: manifests/base/resources/nl-broker/transformers/kustomization.yaml [hunk 3]
```diff
@@ -15,7 +15,11 @@ patches:
         parserUri,
         dpsUri,
         mongoDbUri,
-        keysUri
+        keysUri,
+        ingressClass,
+        secretClientCertificates,
+        hostClientCertificates,
+        pvcClientCertificates,
```
Comment (MoKassem) [inline manifests/base/resources/nl-broker/transformers/kustomization.yaml:22]: Noticed: `nl-broker` need ingressClass and client certs

**Comment by ffoysal** [comment]
need to add
```
        - name: LOG_LEVEL
          value: INFO
```

**Review by ffoysal — APPROVED** [review]

**Pull request merged**